.deps/prod.md

```diff
@@ -0,0 +1,230 @@
+### Product dependencies
+
+| Packages | Resolved CQs |
+| --- | --- |
+| `@babel/code-frame@7.10.1` |  |
+| `@babel/helper-module-imports@7.10.1` |  |
+| `@babel/helper-validator-identifier@7.10.1` |  |
+| `@babel/highlight@7.10.1` |  |
+| `@babel/runtime@7.10.2` |  |
+| `@babel/types@7.10.2` |  |
+| `@eclipse-che/api@7.4.0` |  |
+| `@eclipse-che/workspace-client@0.0.1-1585913592` |  |
```

Comment:
CQ is not needed here because it's eclipse-che part, right?

Reply:
AFAIK, yes

Reply:
Yes. It is.
It can be added as a comment instead of a link for CQ.